packages/gasket-plugin-command/lib/configure.js

```diff
@@ -0,0 +1,37 @@
+// @ts-nocheck
+/* eslint-disable no-unused-vars, no-sync */
+import { applyConfigOverrides } from '@gasket/utils';
+import { gasketBin, processCommand } from './cli.js';
+const isGasketCommand = /\/gasket\.(js|ts|cjs|mjs)$/;
```

Comment:
Thoughts on relaxing this regex a bit for smaller "gasket" situations like `tsx gasket-docs.ts docs`, or `node gasket.some-other-name-part.js my-cmd`?

```suggestion
const isGasketCommand = /gasket[.-\w]*\.(js|ts|cjs|mjs)$/;
```